Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16.0.0
```

Comment:
Alright! Now that you’ve done this can you explain to me what’s happening here? What does this do?